source/includes/fact-non-utf8-data.rst

```diff
@@ -0,0 +1,34 @@
+Compass can have issues displaying collections if documents have 
+invalid UTF8 characters.
+
+If you attempt to query or export this data, the following error 
+message displays: 
```

Comment:
(minimalism -- suggestion)
What if you combined the first two sentences. Something like:
"If you attempt to query or export data with invalid UTF8 characters ..."